proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -53,15 +55,22 @@ public Response hubResponse(@Valid final HubResponseTranslatorRequest hubRespons
             final String entityId = hubResponseTranslatorRequest.getEidasIssuerEntityId().toString();
             final TranslatedHubResponse translatedHubResponse = getAttributesFromVSP(hubResponseTranslatorRequest);
             final CountryMetadataResponse countryMetadata = metatronProxy.getCountryMetadata(entityId);
+            final String assertionConsumerServiceURL = countryMetadata.getAssertionConsumerServices()
```

Comment:
Don't duplicate logic - use the URL pre-selected by the ESP which is in the `hubResponseTranslatorRequest` as `destinationUrl`